index.html

```diff
@@ -1064,111 +1182,129 @@ <h1 class="title is-size-4">Components</h1>
+								</div>
+							</div>
+							<div class="media-content">
+								<h4 class="title is-4">Jane Doe</h4>
+								<div class="subtitle is-6">@jane_doe<span class="visuallyhidden">&nbsp;go to user page</span></div>
+							</div>
+						</div>
+						<div class="content">
+						Laum Ipsum junkah potatoes bookin' it. Moosetown rig up I'm tellin' you way up north bookin' it can't get theyah from heeyah native bean suppah whawf Powrtland Museum of Aht, back woods hawsun around the pit mummah Yessah, mummah muckle riyht on'ta her
             Bean's dinnahbucket bub geez bud sumpin' fierce ayuhpawt Bangah naw, Powrtland Museum of Aht down cellah sumpin' fierce hoppa bub If you can't stand the wintah you don't deserve the summah slower than molasses going uphill in January.
             Sgn'wahl shoggor hrii uaaah R'lyeh uh'e k'yarnak Hastur hupadgh li'hee, hai f'nilgh'ri nilgh'ri n'ghftor ngftaghu vulgtlagln h'hrii throd Nyarlathotep lloig, naflsll'ha nnnsll'ha athg y-ebunma goka chtenff ehyeog cehye. Zhro y'hah
             nogoth phlegeth stell'bsna orr'e ph'Hastur gnaiih throd, uln ya lw'nafh mg nar'luh li'hee wgah'n, sgn'wahl mg nakadishtu chlirgh hupadgh tharanak h'gnaiih.
-            <a>@testio</a>.
-            <a>#css</a>
-            <a>#responsive</a>
-            <br>
-            <time datetime="2018-02-02">12:45 AM - 20 June 2018</time>
-          </div>
-        </div>
-      </div>
-    </div>
-    <button class="modal-close is-large"></button>
-  </div>
-
-  <div id="modal-image1" class="modal modal-fx-3dSlit">
-    <div class="modal-background"></div>
-    <div class="modal-content is-huge is-image">
-      <img src="https://source.unsplash.com/random/?sig=1">
-    </div>
-    <button class="modal-close is-large"></button>
-  </div>
-
-  <div id="modal-image2" class="modal modal-fx-superScaled">
-    <div class="modal-background"></div>
-    <div class="modal-content  is-image">
-      <img src="https://source.unsplash.com/random/?sig=3">
-    </div>
-    <button class="modal-close is-large"></button>
-  </div>
+							<a tabindex="0" role="link">@testio<span class="visuallyhidden">&nbsp;go to user page</span></a>.
+							<a tabindex="0" role="link">#css</a>
+							<a tabindex="0" role="link">#responsive</a>
+							<br>
+							<time datetime="2018-02-02">12:45 AM - 20 June 2018</time>
+						</div>
+					</div>
+				</div>
+			</div>
+			<button class="modal-close is-large" value="close modal">x</button>
+		</div>
+		
+		<div id="modal-image1" class="modal modal-fx-3dSlit">
+			<div class="modal-background"></div>
+			<div class="modal-content is-huge is-image">
+				<img alt="modal random image 1" src="https://source.unsplash.com/random/?sig=1">
+			</div>
+			<button class="modal-close is-large" value="close modal">x</button>
+		</div>
+		
+		<div id="modal-image2" class="modal modal-fx-superScaled">
```

Comment:
Could you please add `role="dialog"` for **<div** (modal)

Reply:
Thank you, I will add role="dialog" in the next commit.